_posts/2019-04-17-sod.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title:  "Apache Flink's Application to Season of Docs"
+date:   2019-4-17 12:00:00
+categories: news
+authors:
+- konstantin:
+  name: "Konstantin Knauf"
+  twitter: "snntrable"
+---
+
+The Apache Flink community is happy to announce its application to the first edition of [Season of Docs](https://developers.google.com/season-of-docs/) by Google. The program is bringing together Open Source projects and technical writers to raise awareness for and improve documentation of Open Source projects. We would like to take this chance to work with one or two technical writers for three months to improve and restructure parts of [our documentation](https://ci.apache.org/projects/flink/flink-docs-stable/) (details below).
```

Comment:
> We would like to take this chance to work with one or two technical writers for three months to improve and restructure parts of

This sounds a bit like we would only like to work with 2 writers in general.
Maybe we should add a sentence somewhere in the text that invites everybody to contribute to the Flink documentation, outside of the SoD program?

Reply:
I changed the first paragraph. What do you think?